sympy/printing/latex.py

```diff
@@ -115,7 +115,7 @@
 
 _between_two_numbers_p = (
     re.compile(r'[0-9][} ]*$'),  # search
-    re.compile(r'[{ ]*[-+0-9]'),  # match
+    re.compile(r'[0-9]'),  # match
```

Comment:
If the regex is just `[0-9]` then isn't that equivalent to `s.isascii() and s.isdecimal()`?

It would seem simpler to use that than a regex.

Reply:
`isascii` returns true for all ascii characters and `isdecimal` returns true for unicode

Reply:
Right, so:
```
if s.isascii() and s.isdecimal():
```

Reply:
Sorry didn't interpret that correctly. Yes, you're right. It will work

Reply:
Why should we worry about unicode?

Reply:
I don't know. I was just suggesting equivalent code.